docs/usage.rst

```diff
@@ -20,21 +20,21 @@ Primary Components
 Currently there are 3 main objects in Switchy for driving
 *FreeSWITCH*:
 
-:py:class:`~switchy.connection.Connection` - a thread safe wrapper around the
+:py:class:`~switchio.connection.Connection` - a thread safe wrapper around the
 `ESL SWIG python package`_'s `ESLConnection`
 
-:py:class:`~switchy.handlers.EventListener` - the type that contains the core
+:py:class:`~switchio.handlers.EventListener` - the type that contains the core
 event processing loop and logic
     - Primarily concerned with observing and tracking the state of
       a single *FreeSWITCH* process
     - Normally a one-to-one pairing of listeners to slave processes/servers
       is recommended to ensure deterministic control.
-    - Contains a :py:class:`~switchy.connection.Connection` used mostly for receiving
+    - Contains a :py:class:`~switchio.connection.Connection` used mostly for receiving
       events only transmitting ESL commands when dictated by :doc:`Switchy apps <apps>`
```

Comment:
Okay, at this point, I think its either forgot to grep for 'Switchy' with a capital S, or you don't care about what I'm pointing out.

Either way, I don't think I need to keep pointing every little issue out anymore.